tests/api/test_api_v2.py

```diff
@@ -335,6 +335,21 @@ def test_sa_post_invalid_ecosystem_value(self):
                                     content_type='multipart/form-data')
         self.assertEqual(response.status_code, 400)
 
+    @patch('bayesian.api.api_v2.StackAnalyses.post_request')
```

Comment:
You may not need to mock `StackAnalyses.post_request` because the control would never go there.